### PR TITLE
Add work aliases

### PR DESCRIPTION
**Pull request by nickmwangemi**
* adds aliases model
* adds aliases to work form
* adds aliases to work overview
* adds aliases to content api

https://www.loom.com/share/f243dca25cdb47789c73c1875d2ae970

### CODE REVIEW

File: indigo_api/models/works.py [hunk 1]
```diff
@@ -903,3 +903,11 @@ def for_abbreviation(cls, abbr):
 def on_subtype_saved(sender, instance, **kwargs):
     # clear the subtype cache
     Subtype._cache = {}
+
+
+class WorkAliases(models.Model):
```
Comment (longhotsummer) [inline indigo_api/models/works.py:908]: Model name must be singular - `WorkAlias`.

File: indigo_api/models/works.py [hunk 2]
```diff
@@ -903,3 +903,11 @@ def for_abbreviation(cls, abbr):
...
+    alias = models.CharField(null=True, blank=True, max_length=255, help_text="Alias e.g. Penal Code, etc")
+    work = models.ForeignKey(Work, on_delete=models.SET_NULL, null=True, related_name="work_aliases")
```
Comment (longhotsummer) [inline indigo_api/models/works.py:910]:
```suggestion
    work = models.ForeignKey(Work, on_delete=models.SET_NULL, null=True, related_name="aliases")
```

File: indigo_api/models/works.py [hunk 3]
```diff
@@ -903,3 +903,11 @@ def for_abbreviation(cls, abbr):
 def on_subtype_saved(sender, instance, **kwargs):
     # clear the subtype cache
     Subtype._cache = {}
+
+
+class WorkAliases(models.Model):
+    alias = models.CharField(null=True, blank=True, max_length=255, help_text="Alias e.g. Penal Code, etc")
```
Comment (longhotsummer) [inline indigo_api/models/works.py:909]:
```suggestion
    alias = models.CharField(null=False, blank=False, max_length=255, help_text="Alias e.g. Penal Code, etc")
```

**Comment by longhotsummer** [comment]
https://www.loom.com/share/f243dca25cdb47789c73c1875d2ae970

**Review by actlikewill — COMMENTED** [review]
I'm seeing the change to the way we load and save formsets. For the commencements_made, saving is slightly different. since we have some saving logic, we call formset.save() method directly.

**Review by actlikewill — APPROVED** [review]
@longhotsummer I've made updates to this PR. I've added a base formset and a base partial form to standardize things a bit

**Comment by longhotsummer** [comment]
> @longhotsummer I've made updates to this PR. I've added a base formset and a base partial form to standardize things a bit

Nice!

**Pull request merged**